web/package.json

```diff
@@ -14,8 +14,9 @@
     "lint:fix": "eslint --ext .ts src --fix",
     "format": "prettier --write \"**/*.{js,ts,css,scss,md,json,yml}\"",
     "format:html": "prettier --write \"**/*.{html,}\"",
-    "e2e": "cypress install && cypress run",
-    "e2e:open": "cypress install && cypress open"
+    "e2e-dev": "cypress install && cypress run",
+    "e2e-dev:open": "cypress install && cypress open",
+    "e2e-standalone": "cypress install && cypress run --config-file cypress-standalone.json"
```

Comment:
did you test all three options?

Regarding `e2e-standalone`. Do you think we should start the REST Service from this location here?

Reply:
I have successfully tested all three options. 
I think we should not start the REST service from the frontend as we then also need to do a maven build first and also we would not delimit the layers and thus create undesired dependencies.